src/ckeditorerror.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+/**
+ * The CKEditor error class.
+ *
+ * All errors will be shortened during the minification process in order to reduce the code size.
+ * Therefore, all error messages should be documented in the same way as those in {@link CKEDITOR.core.log}.
+ *
+ * Read more in the {@link core.log} module.
+ *
+ * @class CKEditorError
+ * @extends Error
+ */
+
+CKEDITOR.define( function() {
+	class CKEditorError extends Error {
```

Comment:
There is no class documentation. We should decide is we document both class and constructor, because they are basically the same.